cdap-ui/app/cdap/services/ThemeHelper.ts

```diff
@@ -419,7 +419,6 @@ function parse1Point0Spec(themeJSON: IOnePoint0SpecJSON): IThemeObj {
     ) {
       features.showWranglerDatamodelViewer = featuresJson['wrangler-datamodel-viewer'];
     }
-
```

Comment:
ditto. let's avoid unnecessary changes.

Reply:
Ok, I will take it into account in the future.